### DISCUSSION TITLE
How to use a venv?

### DISCUSSION THREAD

**Discussion by Yellow-Subroutine** [spakin/SimpInkScr#78] [Q&A]
Hi,

I'am having problems getting this extension to run on my system (Debian testing) as the system python version is now 3.11 and the extension does not seem to be compatible with it.

I also have a working 3.10 installation with which I have created a venv with pillow.

Is there any way to make the extension use this venv?

Many thanks!

**Comment by spakin** [comment]
I'm surprised to hear that a minor update to Python is breaking Simple Inkscape Scripting.  What error message are you seeing?

If Inkscape isn't automatically picking up the Python interpreter that's in your path at the time you launch it, you can edit your Inkscape configuration file to point to a specific executable.  In summary, open your `~/.config/inkscape/preferences.xml` file in an editor, find the `<group id="extensions"…>` stanza, and add a `python-interpreter="/path/to/python3.10"` attribute.  For full details, read Inkscape's [Selecting a specific interpreter version (via preferences file)](https://inkscape.gitlab.io/extensions/documentation/authors/interpreters.html#selecting-a-specific-interpreter-version-via-preferences-file).
